src/pybind/mgr/dashboard/frontend/cypress.json

```diff
@@ -18,5 +18,6 @@
       "mochaFile": "cypress/reports/results-[hash].xml"
     }
   },
-  "retries": 1
+  "retries": 1,
+  "chromeWebSecurity": false
```

Comment:
Its because of the iframe yes and we need to bypass that issue. I went through the [cypress official doc](https://docs.cypress.io/guides/guides/web-security#Disabling-Web-Security) (workarounds are not really applicable for the iframe). Also I searched for a while for some serious consequences and I couldn't find one. Still, I removed the `chromeWebSecurity` from the cypress.json file and limited it to [only use on the cephadm e2e tests.](https://github.com/ceph/ceph/pull/45811/files#diff-888f4c82349ba74e75a1cd79f97e4d365d6b6ca6b4c8e7e2ad77348bf5569252R27)